### PR TITLE
Add DeckTape project

### PR DESCRIPTION
**Pull request by astefanutti**

### CODE REVIEW

File: _posts/documentation/explore/2000-01-07-related-projects.md [hunk 1]
```diff
@@ -67,6 +68,8 @@ Several page capture projects utilizing PhantomJS:
 
  * [Compass Magick](https://github.com/StanAngeloff/compass-magick) is a dynamic image generation for Compass using (among others) PhantomJS.
 
+ * [DeckTape](https://github.com/astefanutti/decktape) is a high-quality PDF exporter from HTML presentation frameworks like impress.js and reveal.js
```
Comment (ariya) [inline _posts/documentation/explore/2000-01-07-related-projects.md]: Why is this duplicated again here?

Comment (astefanutti) [inline _posts/documentation/explore/2000-01-07-related-projects.md]: DeckTape provides a CLI to get screenshots from HTML presentations, similarly to _pageres_, hence the item added in the _Screenshots Utilities_ section. It provides in addition a CLI to export HTML presentations to PDF documents, similarly to _Slippy_, hence the item added in the _Tools_ section.

As I noted that the _pageres_ entry is duplicated to make this kind of distinction, I went for adding two entries. That being said, I would understand that you consider the distinction not worth the duplication.

Comment (ariya) [inline _posts/documentation/explore/2000-01-07-related-projects.md]: Just pick one of the two that is the most suitable/representative. I think it's just fair to everyone to have each tool listed in one category only.

Comment (astefanutti) [inline _posts/documentation/explore/2000-01-07-related-projects.md]: You're right. I've just pushed the updated change.

**Comment by Vitallium** [comment]
Landed! Thank you!

**Pull request merged**